src/modules/admin/components/forms/FormDefinitionTable.tsx

```diff
@@ -10,48 +11,54 @@ import {
   GetFormIdentifiersQuery,
   GetFormIdentifiersQueryVariables,
 } from '@/types/gqlTypes';
-
 export type Row = NonNullable<
   GetFormIdentifiersQuery['formIdentifiers']
 >['nodes'][0];
 
-const columns: ColumnDef<Row>[] = [
-  {
-    header: 'Form Title',
-    render: ({ displayVersion }) => displayVersion.title,
-    width: '300px',
-    linkTreatment: true,
-  },
-  {
-    header: 'Form Type',
-    render: ({ displayVersion }) => <FormTypeChip role={displayVersion.role} />,
-  },
-  {
-    header: 'Applicability Rules',
-    render: ({ displayVersion }) => displayVersion.formRules.nodesCount,
-  },
-  {
-    key: 'system',
-    render: ({ displayVersion }) =>
-      displayVersion.system && (
-        <Chip
-          label='System'
-          size='small'
-          variant='outlined'
-          sx={{ width: 'fit-content' }}
-        />
-      ),
-  },
-  // TODO ADD: # projects active in (based on all rules)
-  // TODO ADD: version
-  // TODO ADD: status
-  // TODO ADD: last updated
-];
-
 interface Props {
   queryVariables: GetFormIdentifiersQueryVariables;
 }
 const FormDefinitionTable: React.FC<Props> = ({ queryVariables }) => {
+  const columns: ColumnDef<Row>[] = useMemo(
+    () => [
+      {
+        header: 'Form Title',
+        render: ({ displayVersion }) => displayVersion.title,
+        width: '300px',
+      },
+      {
+        header: 'Form Type',
+        render: ({ displayVersion }) => (
+          <FormTypeChip role={displayVersion.role} />
+        ),
+      },
+      {
+        key: 'status',
+        header: 'Form Status',
+        render: (identifier) => startCase(identifier.displayVersion.status),
```

Comment:
Added status. This is just the status of the display version, not the full explanatory status. E.g. will show `Published` instead of `Published with unpublished changes`

Reply:
Nice. Maybe it's obvious but could be handy to add this pr comment as a code comment? That could help call our attention to the 2 different statuses if e.g. we ever get a client question about it

Reply:
Good points, added a comment